turbopack/crates/turbopack-ecmascript-runtime/js/src/nodejs/runtime.ts

```diff
@@ -82,7 +82,7 @@ function createResolvePathFromModule(
       strippedAssetPrefix
     );
 
-    return url.pathToFileURL(resolved);
+    return url.pathToFileURL(resolved).href;
```

Comment:
`pathToFileURL` returns a `URL`, not a string like the signature of `createResolvePathFromModule` says. not sure which one is correct -- this was a URL for quite some time i guesss, but i'm guessing a string is what we actually want?